flang/lib/Lower/OpenMP.cpp

```diff
@@ -47,6 +47,18 @@ static void genObjectList(const Fortran::parser::OmpObjectList &objectList,
   }
 }
 
+template <typename Op, typename Terminator>
```

Comment:
You are not using the `Terminator` in the code below. So either use it or remove it from the template.

Reply:
Ah, thanks for catching this. Terminator is created here explicitly.
In my scratch work I first tried with passing it explicitly.